pkg/kotsutil/kots.go

```diff
@@ -445,7 +445,12 @@ func (k *KotsKinds) addKotsKinds(content []byte) error {
 			k.Installation = *decoded.(*kotsv1beta1.Installation)
 		case "kots.io/v1beta1, Kind=HelmChart":
 			if k.HelmCharts == nil {
-				k.HelmCharts = &kotsv1beta1.HelmChartList{}
+				k.HelmCharts = &kotsv1beta1.HelmChartList{
+					TypeMeta: metav1.TypeMeta{
+						APIVersion: "kots.io/v1beta1",
+						Kind:       "HelmChartList",
+					},
+				}
```

Comment:
Instantiate with `apiVersion` and `kind` so that `LoadHelmChartsFromContents(...)` below will work